services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluPaikkaServiceImpl.kt

```diff
@@ -25,47 +29,150 @@ class TormaystarkasteluPaikkaServiceImpl : TormaystarkasteluPaikkaService {
         return luokitteluTulosComplete
     }
 
+    internal fun getKatuluokkaLuokitteluTulos(hanke: Hanke): Luokittelutulos? {
+
+        val katuLuokittelu = mutableListOf<Luokittelutulos>()
+
+        val hankeGeometriatId = hanke.geometriat?.id
+        //if no id let's get out of here
```

Comment:
Tässä pitäisi kyllä oikeastaan lentää InvalidStateException("Hanke.geometriat should be set") tms